_pages/en/open-source-code-form-validation.html

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+ref: formValidation
+lang: en
+permalink: /en/open-source-code-form-validation.html
+---
+
+<h1 property="name" id="wb-cont">Form with validation for open source code</h1>
+<div class="wb-frmvld">
+<form action="#" method="get" id="validation">
+
+<legend><span class="field-name"> Version of the metadata schema</span></legend>
+	<div class="form-group">
+		<label for="shemaVersion" class="required"><span class="field-name">schemaVersion</span> <strong class="required">(required)</strong></label>
+			<select class="form-control" id="shemaVersion" name="shemaVersion" required="required">
+				<option></option>
+				<option value="dr">1.0</option>
+			</select>
+	</div>
+        
+<legend><span class="field-name"> Administration code</span></legend>
+	<div class="form-group">
+		<label for="adminCode" class="required"><span class="field-name">adminCode</span> <strong class="required">(required)</strong></label>
+			<select class="form-control" id="adminCode" name="adminCode" required="required">
+        <option>&nbsp;</option>
+        {% for adminsLevel in site.data.administrations %}
+          {% assign adminsInfo = adminsLevel[1]%}
+          {% for adminInfo in adminsInfo %}
+            <option>{{ adminInfo.code }}</option>
+          {% endfor %}
+        {% endfor %}
+			</select>
+  </div>
+
+
+<legend><span class="field-name">Contact or team for the project</span></legend>
+	<h3>contact: </h3>	
+		<div class="form-group">
+			<label for="urlContact"><span class="field-name">URL</span></label>
+			<input class="form-control" id="urlContact" name="urlContact" type="url" />
+		</div>
+        <div class="form-group">
+			<label for="emailContact" class="required" ><span class="field-name">email </span><strong class="required">(required)</strong></label>
+			<input class="form-control" id="emailContact" name="emailContact" type="email" required="required" />
+		</div>
+        <div class="form-group">
+			<label for="nameContact"><span class="field-name">name</span></label>
+			<input class="form-control" id="nameContact" name="nameContact" type="text" pattern="[A-Za-z\s]" data-rule-lettersonly="true" />
+		</div>
+        <div class="form-group">
+			<label for="phone"><span class="field-name">phone</span> </label>
+			<input class="form-control" id="phone" name="phone" type="tel"  data-rule-phoneUS="true" />
+		</div>
+
+<legend><span class="field-name"> Date of the project</span></legend>
+	<h3>date: </h3>
+		 <div class="form-group">
+			<label for="dateCreated" class="required"><span class="field-name">created</span><strong class="required">(required)</strong><span class="datepicker-format"> (YYYY-MM-DD)</span></label>
```

Comment:
Leave a space between the `</span>` and `</strong`

Lots of fields don't have a space before the (required)